src/index.js

```diff
@@ -7,10 +7,10 @@ var parser = document.createElement('a');
  * @return {Object}                Object with sorted use elements
  */
 function sortItems(items, element) {
-	var xlink = element.getAttribute('xlink:href');
+	var xlink = element.getAttribute('xlink:href') || element.getAttribute('href');
 
-	// return if xlink just contains fragment
-	if (xlink[0] === '#') {
+	// return if no href attribute present or just contains fragment
+	if (!xlink || xlink[0] === '#') {
```

Comment:
Hello @dangoo, thank you! 

I did the check for null or undefined just by adding `!xlink ||` to the if expression. As of the `||` (read it like or-else), the `xlink[0]` part of the expression is not evaluated in case of null, so `!null || null[0]` correctly evaluates without a null error.

Should we do the null check in a separate if expression for better readability?

Reply:
I'm sorry, missed the `!` during testing… 🤦‍♂️
Your new change does not handle `undefined`.
So I'd propose using `if(!Boolean(xlink) || xlink[0] === '#')`.
I'll merge and publish a new version as soon as you changed that.

Reply:
Okay 👌